### PR TITLE
upd(tor): `0.4.6.10` -> `0.4.7.7`

### PR DESCRIPTION
**Pull request by WRM-42**

### CODE REVIEW

**Comment by Henryws** [comment]
At some line you edited, you used some other version of the " character

File: packages/tor/tor.pacscript [hunk 1]
```diff
@@ -1,11 +1,13 @@
 name="tor"
-version="0.4.6.10"
-url="https://dist.${name}project.org/${name}-${version}.tar.gz"
-build_depends="make autoconf"
+repology=("project: "${name}")
```
Comment (Henryws) [inline packages/tor/tor.pacscript:2]:
```suggestion
repology=("project: ${name}")
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
Go through each " character, delete it, and retype "

**Comment by WRM-42** [comment]
I tried to retype them and it doesn't work

**Review by Henryws — APPROVED** [review]

**Pull request merged**